benchmarks/src/swap_ranges.cpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <benchmark/benchmark.h>
+#include <vector>

```

Comment:
Should include `<cstddef>` for `size_t` and `<cstdint>` for `uint8_t`. (If it were just the former, I wouldn't bother with resetting testing.)